### DISCUSSION TITLE
Rows read from CSV are repeating

### DISCUSSION THREAD

**Discussion by sanath-Sunkad** [artilleryio/artillery#1252] [Artillery]
I have 5 rows in each csv and I wanted them to be different row for each request and totally I have 7 POST request. If  I am not wrong there will be 5 vusers created and all of them have to read a different rows from each csv. this is not happening. and total request has to be 35 if each of the vuser complete their scenario. So, how to make each request read different row of CSV sequential or non sequential.

http:
    timeout: 60
  phases:
    - duration: 1
      arrivalRate: 5
      name: Warm up
  payload:
    -
      path: "user.csv"
      fields:
        - "username"
        - "password"
    -
      path: "auth.csv"
      fields:
        - "authusername"
        - "authpassword"
    -
      path: "group.csv"
      fields:
        - "groupName"
        - "groupDescription"
        - "groupMnemonic"

**Comment by hassy** [comment]
hi @sanath-Sunkad 👋  Please see the explanation of `order: sequence` in this thread: https://github.com/artilleryio/artillery/discussions/1261#discussioncomment-1797790

The rows may be repeating because the order in which rows are picked is random, so with only 5 items it's likely that a row will be used more than once.
